src/locutus/api/terminology_mapping.py

```diff
@@ -13,6 +18,15 @@
 class TerminologyMapping(Resource):
     @cross_origin()
     def get(self, id, code):
+        """
+        Retrieves terminology mappings for a given code, optionally including user input details.
+        """
+
+        user_input_param = request.args.get("user_input", default=None)
+        user_id = (

```

Comment:
I recommend we use a [common function ](https://github.com/NIH-NCPI/locutus/blob/bg/fd_1842/src/locutus/api/__init__.py#L26)for this so that we can have a single place to change if we end up going a different route with how we handle this type of data.